components/VC/Views/VCCardView.tsx

```diff
@@ -58,6 +58,7 @@ export const VCCardView: React.FC<
 
   const [fields, setFields] = useState([]);
   const [wellknown, setWellknown] = useState(null);
+  const [isConfigFetched, setIsConfigFetched] = useState(false);
```

Comment:
can't we use wellknown state variable itself to check if it is populated or not ? 

if wellknown != null : 
=> then It is fetched from wellknown
 else 
 => it has not yet populated.